internal/i18n/locales/ja/default.po

```diff
@@ -0,0 +1,142 @@
+msgid ""
+msgstr ""
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#
+# navigation menus
+# ----------------
+
+msgid "nav.issue-code"
+msgstr "コードを発行""
```

Comment:
seems like there might be an extra `"` here

Reply:
thanks, fixed